modules/Forum/pages/panel/forums.php

```diff
@@ -753,6 +759,7 @@
 									'forum_id' => $_GET['forum'],
 									'view' => $view,
 									'create_topic' => $create,
+									'edit_topic' => $create,
```

Comment:
This line should be `'edit_topic' => $edit,`

Reply:
Fixed